app/assets/stylesheets/modules/_timeline.styl

```diff
@@ -191,7 +191,7 @@ button.week__add-block
 
 .week__week-dates
   position relative
-  top 16px
+  top -12px
```

Comment:
If this is going to appear right below the title, I think it would be better to rely on more standard techniques like `margin`. We probably don't need to reposition this anymore.